src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static Random random = new Random();
+
+    public String getRandomColor() {
+        int index = random.nextInt(Color.values().length - 1);
```

Comment:
public int nextInt(int bound)
_Returns a pseudorandom, uniformly distributed {@code int} value
between 0 (inclusive) and the specified value **(exclusive)**_
so, can we get the last enum value if use "-1"?